doc/classes/GPUParticles3D.xml

```diff
@@ -50,6 +50,12 @@
 				Sets the [Mesh] that is drawn at index [code]pass[/code].
 			</description>
 		</method>
+		<method name="update_layer_mask">
+			<return type="void" />
+			<description>
+				Update layer mask of GPUParticles3D
```

Comment:
This should contain a description of what the layer mask does. It should mention that it updates the layer mask that it used with the ``cull_mask`` of Particle Attractors and Colliders to select which Attractors and Colliders interact with this GPUParticles instance.

Reply:
I expanded the description as you said.